CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## [1.4.3]
+
+* Retina integration with VsCode.
+* In-house download replacement implementation.
+* Show-properties page k8s version is now available with deprecated warning.
+* Dependabot updates and bumps.
```

Comment:
nit: Dependent Bot updates*

Reply:
:) its not spelling mistake it shout the real name of the bot :) 

<img width="1024" alt="Screenshot 2024-04-23 at 12 08 36 PM" src="https://github.com/Azure/vscode-aks-tools/assets/6233295/4006aea3-4df2-44a9-a815-853b688d7dd4">